src/lib/ui/core/Notifications/index.ts

```diff
@@ -1,2 +1,34 @@
+import { toast } from 'svelte-sonner'
+
 export { default } from './Notifications.svelte'
-export { toast as notifcation } from 'svelte-sonner'
+import Component from './Notification.svelte'
+import type { ComponentProps, ComponentType } from 'svelte'
+
+function createToast(
+  icon: string,
+  message: string,
+  options?: Omit<ComponentProps<Component>, 'icon' | 'message'>,
+) {
+  return toast.custom(Component as unknown as ComponentType, {
```

Comment:
I had to do this because of the different types of components in Svelte 4 and Svelte 5

Reply:
Here's the error:
```
Argument of type '__sveltets_2_IsomorphicComponent<Props, { closeToast: CustomEvent<any>; } & { [evt: string]: CustomEvent<any>; }, {}, {}, "">' is not assignable to parameter of type 'ComponentType'.
  Type '__sveltets_2_IsomorphicComponent<Props, { closeToast: CustomEvent<any>; } & { [evt: string]: CustomEvent<any>; }, {}, {}, "">' is not assignable to type 'new (options: ComponentConstructorOptions<Record<string, any>>) => SvelteComponent<Record<string, any>, any, any>'.
    Types of parameters 'options' and 'options' are incompatible.
      Type 'ComponentConstructorOptions<Record<string, any>>' is not assignable to type 'ComponentConstructorOptions<Props>'.
        Type 'Record<string, any>' is missing the following properties from type 'Props': icon, messagets(2345)
```